ddtrace/debugging/_function/discovery.py

```diff
@@ -9,7 +9,7 @@
 try:
     from collections.abc import Iterator
 except ImportError:
-    from collections import Iterator  # type: ignore[attr-defined,no-redef]
+    from collections import Iterator  # type: ignore
```

Comment:
is this change needed?

Reply:
No idea at what point it went away... restored.